ftest.sh

```diff
@@ -294,7 +296,8 @@ wq
 EOF
 fi
 
-pushd src/tests/ftest
+#pushd src/tests/ftest
```

Comment:
I would **strongly** suggest you move this dir to a new name (or just `rm -rf` it even) to be sure that nothing you don't know about (yet) is still referencing it or working out of it.

Reply:
Tested with this suggestion, moved this dir to a new one before the test, and moved it back after. It passed all CI tests.